web/src/main/java/org/cbioportal/web/util/StudyViewFilterApplier.java

```diff
@@ -243,6 +258,34 @@ public SampleIdentifier apply(Sample sample) {
         return sampleIdentifiers;
     }
 
+    private List<Integer> extractEntrezGeneIds(List<String> geneIds) {
```

Comment:
For hugoSymbols that cannot be resolved(the gene does not exist), how do we handle the case ?

Reply:
they are ignored